src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -58,8 +58,8 @@ void java_bytecode_languaget::get_language_options(const cmdlinet &cmd)
   object_factory_parameters.string_printable = cmd.isset("string-printable");
   if(cmd.isset("java-max-vla-length"))
     max_user_array_length=std::stoi(cmd.get_value("java-max-vla-length"));
-  if(cmd.isset("lazy-methods-context-sensitive"))
-    lazy_methods_mode=LAZY_METHODS_MODE_CONTEXT_SENSITIVE;
+  if(cmd.isset("symex-driven-lazy-loading"))
```

Comment:
Somewhere there must be `--help` output describing this, which also needs to be fixed. And so does the parse-options header file.

Reply:
Actually no, the option was added ages ago in anticipation of a driver program using it. That is now happening, but the anticipated name isn't the best.

Reply:
So you're removing an option? Different kind of update, but still one expected?!

Reply:
Nobody (except this file) ever accepted it until now. JBMC now accepts "symex-driven-lazy-loading".

Reply:
Ok.